src/jni/java/src/main/java/com/vesoft/nebula/NebulaCodec.java

```diff
@@ -0,0 +1,104 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NebulaCodec {
+    private static final Logger LOGGER = LoggerFactory.getLogger(NebulaCodec.class.getName());
+
+    static {
+        System.loadLibrary("nebula_codec");
```

Comment:
We'd better catch exception/error here, when java client import this as dependency, will throw exception and error.
```
        try {
            System.loadLibrary("nebula_codec");
        } catch (Exception e) {
            LOGGER.error(e.getMessage(), e);
        } catch (Error e) {
            LOGGER.error(e.getMessage(), e);
        }
```

Reply:
Good point